unit_converter/__init__.py

```diff
@@ -0,0 +1,460 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import annotations
+
+import json
+import re
+import traceback
+from datetime import datetime
+from pathlib import Path
+from typing import Any
+from urllib.request import urlopen
+
+import albert
+import inflect
+import pint
+
+default_trigger = "convert "
```

Comment:
try to avoid globals. they will stay in memory on unloading a plugin. if i got pybind docs correctly